src/middleware/router.js

```diff
@@ -776,6 +859,6 @@ function isMethodAllowed (ctx, channel) {
 export async function koaMiddleware (ctx, next) {
   const _route = promisify(route)
   await _route(ctx)
-  await messageStore.storeResponse(ctx, () => {})
+  //await messageStore.storeResponse(ctx, () => {})
```

Comment:
Should this still be commented out? Are we saving the response elsewhere or does this need to come back in at some point?

Reply:
The comment can be removed. 
Without the 'messageStore.storeResponse' call here, the rerunApp, tcpApp and pollingApp functions won't be able to update transactions until we refactor them.

Reply:
Ok cool so for the new changes we are doing this isnt needed. but for the rerun/tcp/polling apps this will still be needed (until we refactor).

Lets maybe just add a comment here as to why this is commented out, and that we need to address this for the other applications served via Koa